packages/ui-devkit/src/compiler/types.ts

```diff
@@ -218,6 +227,17 @@ export interface UiExtensionCompilerOptions {
      * @default 4200 | undefined
      */
     watchPort?: number;
+
+    /**
+     * @description
+     * Allows to pass additional arguments for watcher (in development mode) and compiler (in production mode).
+     * 
```

Comment:
I'd make it explicit that these are being passed to the `ng build` or `ng serve` commands and include a link:

---
```
Additional command-line arguments which will get passed to the [ng build](https://angular.io/cli/build) 
command (or [ng serve](https://angular.io/cli/serve) if `devMode = true`).
```

Also, please add the following tag to show when this was introduced:

```
@since 1.5.0
```